linux/LinuxProcess.c

```diff
@@ -231,6 +233,8 @@ static void LinuxProcess_rowWriteField(const Row* super, RichString* str, Proces
    switch (field) {
    case CMINFLT: Row_printCount(str, lp->cminflt, coloring); return;
    case CMAJFLT: Row_printCount(str, lp->cmajflt, coloring); return;
+   case GPU_PERCENT: Row_printPercentage(lp->gpu_percent, buffer, n, 5, &attr); break;
+   case GPU_TIME: xSnprintf(buffer, n, "%*llu", Row_fieldWidths[GPU_TIME], lp->gpu_time); break;
```

Comment:
Do we really need to print the GPU time with that much precision?
I was thinking if the `humanTimeUnit()` format can also apply here. Maybe this printing format could become a new function called `Row_printNanosecends()`.